#request
int32 address
int32 multiplex
---
#response
bool success
